packages/contracts-bedrock/src/Safe/DeputyGuardianModule.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Safe } from "safe-contracts/Safe.sol";
+import { Enum } from "safe-contracts/common/Enum.sol";
+
+import { SuperchainConfig } from "src/L1/SuperchainConfig.sol";
+import { OptimismPortal2 } from "src/L1/OptimismPortal2.sol";
+import { IDisputeGame } from "src/dispute/interfaces/IDisputeGame.sol";
+import { ISemver } from "src/universal/ISemver.sol";
+
+import "src/libraries/DisputeTypes.sol";
+
+/// @title DeputyGuardianModule
+/// @notice This module is intended to be enabled on the Security Council Safe, which will own the Guardian role in the
+///         SuperchainConfig contract. The DeputyGuardianModule should allow a Deputy Guardian to administer any of the
+///         actions that the Guardian is authorized to take. The security council can revoke the Deputy Guardian's
+///         authorization at any time by disabling this module.
+contract DeputyGuardianModule is ISemver {
+    /// @notice Error message for unauthorized access
+    error OnlyDeputyGuardian();
+
+    /// @notice Error message for failed transaction execution
+    error TransactionExecutionFailed(string);
+
+    /// @notice Emitted when the SuperchainConfig is paused
+    event SuperchainConfigPaused();
```

Comment:
Thoughts on trying to use more common errors across the codebase? ie just use `Paused()` so that the same abi can come from many contracts to simplify things

Reply:
For simplicity, I will just redefine the same event, so that we can consolidate them into one place in the future. It already means I can avoid redeclaring it in the test suite.